### PR TITLE
Align dates deadlines page with recruitment process

### PR DESCRIPTION
**Pull request by MylesJarvis**
## Context

Our [dates and deadlines page](https://www.apply-for-teacher-training.service.gov.uk/provider/service-guidance/dates-and-deadlines) is out of date. We created a new page, [the application process page](https://www.apply-for-teacher-training.service.gov.uk/candidate/about-the-teacher-training-application-process), which is candidate focussed and does a similar job. This brings both pages into better alignment.

Q: I've removed copy that we had sitting in 2024-2025 about 30 working days etc. Should we include this in 'current' cycle copy now?

## Changes proposed in this pull request

Before:
image

After: 
image

## Guidance to review

Are we happy the dates for next cycle are now correct (we can always amend, but they should at least look right on first glance, unlike they do now).

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by MylesJarvis** [comment]
Sorted timetable dates separately now - https://github.com/DFE-Digital/apply-for-teacher-training/pull/8632. I'll rebase once into main.

File: app/views/content/dates_and_deadlines.md [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
-| <%= holidays[:easter][:begins].to_fs(:day_and_month) %> to <%= holidays[:easter][:ends].to_fs(:govuk_date) %> | This period is not counted as working days when calculating time to make a decision. |
-| <%= CycleTimetable.show_summer_recruitment_banner.to_fs(:govuk_date) %> | Time to make a decision is reduced from 40 working days to 20 working days. |
-| <%= CycleTimetable.apply_1_deadline.to_fs(:govuk_date_and_time) %> | Candidates can no longer apply for courses, unless they have already applied within this recruitment cycle. |
-| <%= CycleTimetable.apply_2_deadline.to_fs(:govuk_date_and_time) %> | Candidates can no longer apply for courses. |
-| <%= CycleTimetable.reject_by_default.to_fs(:govuk_date_and_time) %> | End of <%= RecruitmentCycle.cycle_name(RecruitmentCycle.current_year) %> recruitment cycle - applications awaiting decisions are automatically rejected. |
+| <%= CycleTimetable.apply_opens.to_fs(:govuk_date_and_time) %> | Start of <%= RecruitmentCycle.cycle_name(RecruitmentCycle.current_year) %> recruitment cycle. Candidates can apply for courses. |
```
Comment (MaeveRoseveare) [inline app/views/content/dates_and_deadlines.md:4]: We're missing the date for Find opening above Apply opening.

**Review by MaeveRoseveare — COMMENTED** [review]
Looks good, we're just missing the date for Find opening on the first table.

File: app/views/content/dates_and_deadlines.md (hunk 1 above)
Comment (MylesJarvis) [reply app/views/content/dates_and_deadlines.md:5]: Pushed

**Pull request merged**